src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
-// write code here
+const tableBody = document.querySelector('table > tbody');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const container = document.querySelector('.container');
+
+container.addEventListener('click', e => {
+  const item = e.target.closest('.button');
+
+  if (!item) {
+    return;
+  }
+
+  const rows = document.querySelectorAll('table > tbody > tr');
+
+  const maxRow = 10;
+  const minRow = 2;
+  const maxColumn = 10;
+  const minColumn = 2;
```

Comment:
Yes, good decision to create separate constants, but notice that they should be in uppercase `MAX_ROW`, `MIN_ROW`, ...

Reply:
My fail, it's can be in camelCase:
> What about all const variables? - This is unnecessary, so uppercasing should not be used for constants within a file. It should be used for exported constants however.